pkg/support/runsupportbundle.go

```diff
@@ -179,6 +140,21 @@ func (e *EksaDiagnosticBundle) PrintBundleConfig() error {
 	return nil
 }
 
+func (e *EksaDiagnosticBundle) WriteBundleConfig() error {
```

Comment:
Is it possible to pass this to the binary through stdin to avoid writing to disk?

Reply:
uh potentially, I'll take that as a follow-on